backpack/extensions/module_extension.py

```diff
@@ -60,6 +60,10 @@ def backpropagate(self, ext, module, g_inp, g_out, bpQuantities):
         warnings.warn("Backpropagate has not been overwritten")
 
     def apply(self, ext, module, g_inp, g_out):
+        """
+        Fetch backpropagated quantities from module output, apply backpropagation
+        rule, and attach the result to module input(s).
+        """
```

Comment:
I will add better docstrings and type hints to this class anyway. So feel free to leave as it is.

Reply:
Leaving as is. Marking as resolved.